recipe/meta.yaml

```diff
@@ -32,15 +32,8 @@ requirements:
     - packaging
     - panel >=0.13.1
     - pandas >=0.20.0
-    - param >=1.9.3,<2.0
+    - param >=1.12.0,<3.0
     - pyviz_comms >=0.7.4
-    # Notebook dependencies
-    - ipython >=5.4.0
-    - notebook
-    # Recommended dependencies: IPython Notebook + pandas + matplotlib + bokeh
-    # It seems holoviews isn't compatible with bokeh >=3.0.0
-    - bokeh >=2.4.3,<3.0
-    - matplotlib-base >=3
```

Comment:
On conda (and conda-forge), we have historically afaik had both `bokeh` and `matplotlib-base` as dependencies.

Reply:
Happy to readd `matplotlib-base`.